string name
uint8 frame_id
uint8 height_id
iroc_mission_handler/Waypoint[] points
uint8 terminal_action

